src/pages/books/Books.jsx

```diff
@@ -1,21 +1,14 @@
-import React, { useState } from 'react';
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import BookCard from '../../components/BookCard';
 import Style from './Books.module.css';
 import BookForm from '../../Forms/book-form/BookForm';
+import { removeBook, addBook } from '../../redux/books/booksSlice';
 
 /*  eslint-disable react/jsx-curly-brace-presence, no-unused-vars */
```

Comment:
- [ ] As required by Microverse you are not meant to use any linter `disable rule` in your codebase, the linters are there to make sure your code follows the best practices. I'll suggest you remove the linters disable rule and fix the errors. :+1: :100: